Admin/DashboardAdmin.php

```diff
@@ -139,8 +146,16 @@ protected function configureSideMenu(MenuItemInterface $menu, $action, AdminInte
             array('uri' => $admin->generateUrl('compose', array('id' => $id)))
         );
 
-        $menu->addChild('sidemenu.link_list_blocks',
-            array('uri' => $admin->generateUrl('sonata.dashboard.admin.dashboard|sonata.dashboard.admin.block.list', array('id' => $id)))
-        );
+        $menu->addChild('sidemenu.link_render_dashboard', array(
+            'uri' => $admin->generateUrl('render', array(
+                'id' => $id,
+            )),
```

Comment:
I think `generateMenuUrl` is exactly what we are looking for: https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Route/DefaultRouteGenerator.php#L115
But the naming is reaaaaally shitty.